foundation-extensions/profile/profile-api-references.md

```diff
@@ -811,30 +995,36 @@ UserProfile.updateUserAttributes(attributeDict: profileMap)
 
 ### Objective-C
 
+**Syntax**
+
+```objectivec
++ (void)updateUserAttributesWithAttributeDict:(NSDictionary<NSString *,id> * _Nonnull)
```

Comment:
Same comment here, the example does not match this syntax

Reply:
The example code is fixed.